src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -126,6 +126,10 @@ if (!class_exists('SymfonyBlacklistPhpunit', false)) {
 if (class_exists('PHPUnit_Util_Blacklist')) {
     PHPUnit_Util_Blacklist::$blacklistedClassNames['SymfonyBlacklistPhpunit'] = 1;
     PHPUnit_Util_Blacklist::$blacklistedClassNames['SymfonyBlacklistSimplePhpunit'] = 1;
+} elseif (method_exists('PHPUnit\Util\Blacklist', 'addDirectory')) {
+    (new PHPUnit\Util\BlackList())->getBlacklistedDirectories();
+    PHPUnit\Util\Blacklist::addDirectory(\dirname(\dirname((new \ReflectionClass('SymfonyBlacklistPhpunit'))->getFileName())));
+    PHPUnit\Util\Blacklist::addDirectory(\dirname(\dirname((new \ReflectionClass('SymfonyBlacklistSimplePhpunit'))->getFileName())));
```

Comment:
one "dirname" for both here (see the value: `['SymfonyBlacklistPhpunit'] = 1;`, vs `2` in the other file)